Felienne/hedy
# [UI IMPROVEMENT] Adds notifier if user doesn't have a public profile yet

**Description**
In this PR we make a small QoL change where we add a notification and explanation of a user doesn't have a public profile yet but visits the highscores page.

**Fixes**
This PR fixes #2997.

**How to test**
Navigate to the highscores page with and without a public profile. Verify that the notification is only shown if no public profile exists. Also verify that the button works and automatically opens the public profile section when clicked.
